env.arolla.sh

```diff
@@ -133,15 +133,15 @@ setCppEnvironment()
     cat > $ENVIRONMENT_TEMPFILE <<- EOF
         # Generated with the build script
         # implicit module purge
-        module load cmake/3.14.3
         module load craype-x86-skylake
         module load craype-network-infiniband
         module load slurm
         # Gnu env
         module load PrgEnv-gnu
         module switch mvapich2/2.3.1-gcc-7.4.0-2.31.1 mvapich2/2.2-gcc-7.4.0-2.31.1-cuda-9.2
         module load cuda92/toolkit/9.2.88 craype-accel-nvidia70
-        module load cmake
```

Comment:
there seem to be no cmake now. intended?

Reply:
it is below, line 143. Just the version provided by modules is not enough for GT, unfortunately

Reply:
Ah yes, I was just scanning for a module. Here would be one `/users/jenkins/easybuild/arolla-ln/modules/all/cmake/3.14.5`.

Reply:
uhmm yes, I thought I tried that one, but now I see that GT requires exactly 3.14.5.... I ll try

Reply:
done